docs/src/spark-usage/spark-job-object-store.md

```diff
@@ -0,0 +1,48 @@
+# Spark job using Data in Object Store
+
+In DataLabs, users can create Spark clusters from the notebooks in
+their sessions and submit jobs to them. This page describe how a
+user can submit a Spark job using data stored in the Object Storage.
+
+## Prerequisites
+
+The user would need to have first obtained the Object store credentials
+from the JASMIN support team. These would be in the form of two
+strings - an access key and a secret key. These should be stored securely.
+
+## Create Notebook
+
+Create a Jupyter notebook within your defined project.
+When the notebook is created the Spark properties needed
+for the Spark cluster to be created is dynamically loaded into the notebook environment.
+
+### Create Spark session and load credentails
+
+```python
+from pyspark.sql import SparkSession
+spark = SparkSession.builder.appName('Cropnet') \
+    .config("fs.s3a.access.key", "xxxxxxxxx") \
```

Comment:
Should we add warnings here about securing access keys and secrets? In this style they will be saved to disk with the notebook and so available to all other users of the same drive. Might be better to see if there is a way to set them in the environment?

Reply:
Updated the PR

Reply:
Hi @iwalmsley kindly have a look at this PR

Reply:
Tested this and working well for me.